core/templates/dev/head/pages/landing-pages/topic-landing-page/topic-landing-page.constants.ts

```diff
@@ -27,11 +27,11 @@ export class TopicLandingPageConstants {
         collection_id: '4UgTQUc1tala',
         page_data: {
           image_1: {
-            file_name: 'matthew_paper.png',
+            file_name: 'matthew_paper.webp',
```

Comment:
Probably use just the `matthew_paper` and then generarte both the `webp` and `png` versions in the place where these data are used.